book/content/make/make.md

```diff
@@ -694,6 +705,102 @@ The published Makefile in the repository does not contain the paper, but this
 collaboration as only a single repository needed to be shared that contains 
 the code, the results, and the manuscript.
 
+## Including numerical results and tables
+
+At this point you may be thinking "That's so cool that I can now include figures
+into my manuscripts! But how exactly does this work for numerical results?"
+
+The reproducible paper linked above shows one way of doing it:
+After the results are computed, they are written out in the form of a LaTeX
+table.
+Here is how one of these tables looks like right after it was computed:
+
+```
```

Comment:
You could enable syntax highlighting for these code snippets by using ```latex

Reply:
ha, thanks! I wasn't aware of this flavor of highlighting :)